src/step.rs

```diff
@@ -664,7 +664,7 @@ impl Step {
             WslUpdate =>
             {
                 #[cfg(windows)]
-                runner.execute(*self, "WSL", || windows::update_wsl(ctx))?
+                runner.execute(*self, "WSL Update", || windows::update_wsl(ctx))?
```

Comment:
Should match separator
```suggestion
                runner.execute(*self, "Update WSL", || windows::update_wsl(ctx))?
```

Reply:
Ah I forgot about the separator text. Will fix.